Sources/SwiftDocCUtilities/PreviewServer/RequestHandler/FileRequestHandler.swift

```diff
@@ -157,13 +157,17 @@ struct FileRequestHandler: RequestHandlerFactory {
                 
                 // Read the file contents
                 do {
-                    data = try Data(contentsOf: fileURL, options: .mappedIfSafe)
```

Comment:
very minor: Is the a memory mapped option we can use here?

Reply:
not yet, but there's a PR proposed for this https://github.com/apple/swift-system/pull/68